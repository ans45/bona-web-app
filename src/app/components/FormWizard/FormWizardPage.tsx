/**
 *
 * FormWizardPage
 *
 */
import React, { ReactNode } from 'react';
import { ValidationErrors } from 'final-form';

export interface Props {
  children?: ReactNode;
  validate: (
    Object,
  ) => ValidationErrors | Promise<ValidationErrors> | undefined;
}

export const FormWizardPage = ({ children }: Props) => {
  return <>{children}</>;
};
