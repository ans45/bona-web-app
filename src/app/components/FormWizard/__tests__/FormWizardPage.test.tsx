import React from 'react';
import { render } from '@testing-library/react';
import { FormWizardPage } from '../FormWizardPage';

describe('<FormWizardPage  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <FormWizardPage validate={() => undefined}>
        <h1>Hello World!</h1>
      </FormWizardPage>,
    );

    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
