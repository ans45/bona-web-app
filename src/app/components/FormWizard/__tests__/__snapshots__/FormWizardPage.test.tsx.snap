// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWizardPage  /> should match snapshot 1`] = `
<h1>
  Hello World!
</h1>
`;
